migrations/versions/rev-2019-09-15-08:29:32-d1c2b8711223_.py

```diff
@@ -0,0 +1,39 @@
+"""empty message
+
+Revision ID: d1c2b8711223
+Revises: 7c32ba647a18
+Create Date: 2019-09-15 08:29:32.373041
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'd1c2b8711223'
+down_revision = '7c32ba647a18'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('events', 'is_billing_info_mandatory',
+               existing_type=sa.BOOLEAN(),
+               nullable=False)
+    op.alter_column('orders', 'is_billing_enabled',
+               existing_type=sa.BOOLEAN(),
+               server_default=false
+               nullable=False)
```

Comment:
Why do you send a PR without testing?

This is not even valid Python syntax

Reply:
I missed a comma.

Reply:
And do you think this is valid python? `server_default=false`

Reply:
corrected it

Reply:
Still wrong